docs/upgrade-guide.md

```diff
@@ -52,6 +52,10 @@ All our layers enable depth test so we are going set this state during initializ
 For any custom needs, these parameters can be overwritten by updating them in [`onWebGLInitialized`](docs/api-reference/react/deckgl.md#onWebGLInitialized) callback or by passing them in `parameters` object to `drawLayer` method of `Layer` class.
 
 
+### assembleShaders
+
+`assembleShaders` was deprecated in v4.1 and now removed in v5. As described in v4.1 upgrade guide please use `Model` class instead.
```

Comment:
I would rephrase this:

The `assembleShaders` function was moved to luma.gl in v4.1 and is no longer re-exported from deck.gl. As described ...., or import it from luma.gl where it is defined.x